flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionITCase.java

```diff
@@ -1229,6 +1242,11 @@ private CastTestSpecBuilder fromCase(DataType dataType, Object src, Object targe
             this.columnTypes.add(dataType);
             this.columnData.add(src);
             this.expectedValues.add(target);
+            assertThat(
+                            LogicalTypeCasts.supportsExplicitCast(
```

Comment:
The formatting here, seems a bit off, but might be wrong.

Reply:
It is not :rofl: